mediathread/urls.py

```diff
@@ -214,10 +214,15 @@
     # Composition Space
     (r'^project/', include('mediathread.projects.urls')),
 
-    # Instructor Dashboard & reporting
+    # Instructor Dashboard
     url(r'^course/(?P<pk>\d+)/dashboard/$',
```

Comment:
The course prefix here is a bit problematic due to the way Courseaffils middleware is activated. Right now, "course" is considered an anonymous path, and as such the middleware does not decorate the request with the course. The base template and LoggedInCourseMixin currently expect request.course to be available.

If you're working on eliminating the decoration (yay!), you'll need to look at assumptions in the base templates and the mixins.